sass/themes/schemas/components/light/_combo.scss

```diff
@@ -279,12 +279,25 @@ $bootstrap-combo: extend(
 /// @prop {Color} clear-button-background-focus [transparent] - The combo clear button background color when the combo is focused.
 /// @prop {Map} clear-button-foreground [color: ('gray', 600)] - The combo toggle clear foreground color.
 /// @prop {Map} clear-button-foreground-focus [color: ('gray', 600)] - The combo clear button foreground color when the combo is focused.
+/// @prop {Map} toggle-button-background [color: transparent] - The combo toggle button background color.
+/// @prop {Map} toggle-button-background-focus [transparent] - The combo toggle button background color when the combo is focused.
+/// @prop {Map} toggle-button-background-focus--border [transparent] The combo toggle button background color when the combo is focused.
+/// @prop {Map} toggle-button-background-disabled [transparent] - The combo toggle button background color when the combo is disabled.
+/// @prop {Map} toggle-button-foreground [color: ('gray', 700)] - The combo toggle button foreground color.
+/// @prop {Map} toggle-button-foreground-focus [color: ('gray', 700)] - The combo toggle button foreground color when the combo is focused.
+/// @prop {Map} toggle-button-foreground-filled [color: ('gray', 900)] - The combo toggle button foreground color when the combo is focused.
```

Comment:
toggle-button-foreground,
toggle-button-foreground-focus,
toggle-button-foreground-filled
Those are not used in this schema, you can remove them